.github/workflows/build_nilrt.yml

```diff
@@ -147,6 +147,20 @@ jobs:
         SystemTestsRunner
         test_mutual_tls_config.json
         UnitTestsRunner
+        -C ${GITHUB_WORKSPACE}
+        LICENSE
+        ThirdPartyNotices.txt
+
+    - name: Tar Release Build Files
+      if: ${{ startsWith(github.ref, 'refs/heads/releases') }}
+      run: >-
+        tar -cvf NILinuxRealTime-x86_64.tar
```

Comment:
Might be nice to rename the artifacts to exactly match the release names:

Linux Desktop: 
ni-grpc-device-server-linux-glibc2_23-x64.tar.gz  (created with "tar -cvzf")

LinuxRT: 
ni-grpc-device-server-ni-linux-rt-x64.tar.gz   (created with "tar -cvzf")

Windows: 
ni-grpc-device-server-windows-x64.zip

Reply:
Done